sdk/include/aditof/camera_definitions.h

```diff
@@ -36,6 +36,8 @@
 #include "frame_definitions.h"
 #include "status_definitions.h"
 
+#include "../../sdk/src/cameras/itof-camera/tofi/tofi_camera_intrinsics.h"
```

Comment:
This is not ok. tofi_camera_intrinsics.h is not part of the public API. Client will get an error with missing file.